CRM/Event/Tokens.php

```diff
@@ -60,22 +60,86 @@ protected function getBespokeTokens(): array {
         'data_type' => 'String',
         'audience' => 'user',
       ],
-      'contact_email' => [
+      'loc_block_id.email_id.email' => [
         'title' => ts('Event Contact Email'),
-        'name' => 'contact_email',
```

Comment:
do we need to put an upgrade status message or something about these tokens not working now?

Reply:
they should still work cost they are in the deprecated mapping